### ISSUE TITLE
Host XLS standards as HTML files for better discoverability and design

### ISSUE THREAD

**Issue by intelliot** [XRPLF/xrpl-dev-portal#1779]
[XRPL Standards](https://github.com/XRPLF/XRPL-Standards) are harder to discover, and don't look as nice, versus pages on XRPL.org.

For better discoverability and design (than reading on GitHub), we could:

1. Fetch Markdown files from https://github.com/XRPLF/XRPL-Standards
2. Host them somewhere under XRPL.org

The result would be comparable to https://eips.ethereum.org/

**Comment by mDuo13** [comment]
I think having xls.xrpl.org show nice HTML versions of standards—potentially including supplemental information in separate files—would be nice, and should be plenty doable using GitHub Pages and a GH Action that runs on commit to master. We could use Jekyll, since GH supports that natively, or any other static site generator (e.g. currently XRPL.org uses Dactyl)

**Comment by amarantha-k** [comment]
- +1 to hosting XLS standards as HTML files. 
- Supplemental information (such as documentation/tutorials) for each proposal will need to be updated frequently during development and hence may be better for contributors to have autonomy on source location/hosting.

**Comment by alloynetworks** [comment]
The conversations are almost as important as the XLS itself. What is the plan to keep them in sync? I’m not in favor of these coming across as a static document.

**Comment by mvadari** [comment]
I think it'd be fine to have the XLSes on xrpl.org, but only after they're merged into the Standards repo via PR (i.e. have gotten some measure of consensus).

**Comment by mDuo13** [comment]
Now that xls.xrpl.org is available, I think this can be marked as done. (Thanks @mvadari )
